### PR TITLE
Copy KubeOne master docs to v1.0

### PR DESCRIPTION
**Pull request by xmudrii**
Fixes https://github.com/kubermatic/kubeone/issues/982

/assign @kron4eg

### CODE REVIEW

File: content/kubeone/v1.0/compatibility_info/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+* RHEL 8
+* CoreOS
+* Flatcar
+
+\*\* Only Kubernetes versions 1.18.6 and 1.17.9 are known to work properly with
+CentOS 7.
+
+[upstream-supported-versions]: https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions
+[kubernetes-issue-93194]: https://github.com/kubernetes/kubernetes/issues/93194
+[terraform-configs]: https://github.com/kubermatic/kubeone/tree/master/examples/terraform
```
Comment (kron4eg) [inline content/kubeone/v1.0/compatibility_info/_index.en.md:78]:
```suggestion
[terraform-configs]: https://github.com/kubermatic/kubeone/tree/release/v1.0/examples/terraform
```

Now I'm not saying we should do this, but we might want to, let's discuss.

File: content/kubeone/v1.0/getting_kubeone/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+```
+
+## Building KubeOne
+
+If you have the Go toolchain configured, you can use `go get` to obtain KubeOne.
+
+To get the latest stable release:
+
+```
+GO111MODULE=on go get k8c.io/kubeone@master
```
Comment (kron4eg) [inline content/kubeone/v1.0/getting_kubeone/_index.en.md:82]:
```suggestion
GO111MODULE=on go get k8c.io/kubeone
```

File: content/kubeone/v1.0/compatibility_info/_index.en.md (hunk 1 above)
Comment (xmudrii) [reply content/kubeone/v1.0/compatibility_info/_index.en.md:78]: I think that master docs should point to the master branch, but it makes sense to change this for the v1.0 docs.

Comment (xmudrii) [reply content/kubeone/v1.0/compatibility_info/_index.en.md]: Changed links for the v1.0 docs to point to the `release/v1.0` branch.

File: content/kubeone/v1.0/getting_kubeone/_index.en.md (hunk 2 above)
Comment (xmudrii) [reply content/kubeone/v1.0/getting_kubeone/_index.en.md]: Fixed.

**Pull request merged**